iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -1500,6 +1512,412 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
+		})
+	})
+
+	Describe(`ReplaceTestActionControlTemplateVersion - Update an action control template version`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+
+		It(`ReplaceTestActionControlTemplate(replaceActionControlTemplateOptions *ReplaceActionControlTemplateOptions)`, func() {
+
+			templateActionControl := &iampolicymanagementv1.TemplateActionControl{
+				ServiceName: core.StringPtr("am-test-service"),
+				Description: core.StringPtr("am-test-service service actionControl"),
+				Actions:     []string{"am-test-service.test.delete", "am-test-service.test.create"},
+			}
+
+			replaceActionControlTemplateVersionOptions := &iampolicymanagementv1.ReplaceActionControlTemplateOptions{
+				ActionControl:           templateActionControl,
+				Description:             core.StringPtr("Test update of ActionControl Template from GO SDK"),
+				Committed:               core.BoolPtr(true),
+				IfMatch:                 core.StringPtr(testActionControlTemplateVersionETag),
+				Version:                 core.StringPtr(testActionControlUpdateTemplateVersion),
+				ActionControlTemplateID: &testActionControlTemplateID,
+			}
+
+			actionControlTemplate, response, err := service.ReplaceActionControlTemplate(replaceActionControlTemplateVersionOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(actionControlTemplate).ToNot(BeNil())
+			Expect(actionControlTemplate.AccountID).To(Equal(core.StringPtr(testTargetEnterpriseID)))
+			Expect(actionControlTemplate.Name).To(Equal(core.StringPtr(exampleActionControlTemplateName)))
+			Expect(actionControlTemplate.State).To(Equal(core.StringPtr("active")))
+
+			testActionControlUpdateTemplateVersion = *actionControlTemplate.Version
+		})
+	})
+
+	Describe(`ListActionControlTemplates - Get action control templates by attributes`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ListActionControlTemplates(listActionControlTemplatesOptions *ListActionControlTemplatesOptions)`, func() {
+			listActionControlTemplatesOptions := &iampolicymanagementv1.ListActionControlTemplatesOptions{
+				AccountID:      &testTargetEnterpriseID,
+				AcceptLanguage: core.StringPtr("default"),
+			}
+
+			actionControlTemplateCollection, response, err := service.ListActionControlTemplates(listActionControlTemplatesOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(actionControlTemplateCollection).ToNot(BeNil())
+
+			// Expect(actionControlTemplateCollection.ActionControlTemplates[0].Policy.Type).ToNot(BeNil())
+			Expect(actionControlTemplateCollection.ActionControlTemplates[0].AccountID).To(Equal(&testTargetEnterpriseID))
+			Expect(actionControlTemplateCollection.ActionControlTemplates[0].State).To(Equal(core.StringPtr("active")))
+		})
+	})
+
+	Describe(`GetActionControlTemplate - Retrieve latest action control template version by template ID`, func() {
```

Comment:
Same as other integration tests that are overlapping with the basic case.

Reply:
Added an additional test case to cover changes in the response.